carspeed.py

```diff
@@ -225,7 +276,7 @@ def draw_rectangle(event,x,y,flags,param):
     # dilate the thresholded image to fill in any holes, then find contours
     # on thresholded image
     thresh = cv2.dilate(thresh, None, iterations=2)
-    (_, cnts, _) = cv2.findContours(thresh.copy(), cv2.RETR_EXTERNAL,cv2.CHAIN_APPROX_SIMPLE)
+    cnts, _ = cv2.findContours(thresh.copy(), cv2.RETR_EXTERNAL,cv2.CHAIN_APPROX_SIMPLE)
```

Comment:
I believe this needs to be "_, cnts, _ =" for it to work without a "too many values to unpack" error.